### PR TITLE
Patch for the PR 8754, fixes #9011

### LINKED ISSUE #9011: Existing company record address not converted on upgrade
This PR - https://github.com/frappe/erpnext/pull/8754 - got rid of the address text field and moved to a related address record. While that is great, the upgrade process not take into account existing values.

Old (I obfuscated the address there on purpose):
image

New:
image

I would have expected the upgrade process to have parsed the address and created an associated address record on upgrade. Losing data during an upgrade is not a good practice.

### PR DESCRIPTION
**Pull request by manassolanki**
patch for removing the address field from the company and creating an address doc.

fixes https://github.com/frappe/erpnext/issues/9011

### CODE REVIEW

File: erpnext/patches/v8_0/create_address_doc_from_address_field_in_company.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	# new field address_html is created in place of address field for the company's address
+	# patch for moving the address details in the address doc
+	company_list = frappe.db.sql('''select name, address from `tabCompany` where address is not null''', as_dict=1)
```
Comment (mbauskar) [inline erpnext/patches/v8_0/create_address_doc_from_address_field_in_company.py]: @manassolanki,

Please check if the address column is available or not then use `frappe.db.sql`

**Review by mbauskar — CHANGES_REQUESTED** [review]

**Comment by mbauskar** [comment]
please push the changes and reopen the PR

**Comment by nabinhait** [comment]
Create the address record only if there is already no address record for the same company.

To check address column in the table, you can use `if 'address' in frappe.db.get_table_columns('Company')`.

**Comment by manassolanki** [comment]
@nabinhait ,
This is the patch for the PR https://github.com/frappe/erpnext/pull/8754

**Pull request merged**